<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="CodeIgniter Application Test Suite" tests="37" assertions="35" failures="10" errors="1" time="5.593839">
    <testsuite name="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" fullPackage="Lapangan" tests="15" assertions="25" failures="4" errors="1" time="1.779496">
      <testcase name="test_index" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="21" assertions="3" time="0.143759"/>
      <testcase name="test_sewajadwal" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="29" assertions="3" time="0.100609"/>
      <testcase name="test_sewajadwal_gagal" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="37" assertions="1" time="0.094514"/>
      <testcase name="test_detail" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="43" assertions="3" time="0.113210"/>
      <testcase name="test_inputsewa" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="49" assertions="3" time="0.111328"/>
      <testcase name="test_inputsewa_gagal" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="57" assertions="1" time="0.097984"/>
      <testcase name="test_signup" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="64" assertions="1" time="0.105337"/>
      <testcase name="test_createsewa_sukses" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="74" assertions="1" time="0.208604">
        <error type="PHPUnit_Framework_Exception">Lapangan_test::test_createsewa_sukses
PHPUnit_Framework_Exception: Argument #2 of PHPUnit_Framework_Assert::assertContains() must be a array, iterator or string

C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php:83
</error>
      </testcase>
      <testcase name="test_ceklogin" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="86" assertions="1" time="0.116879"/>
      <testcase name="test_login_sukses" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="96" assertions="0" time="0.106674">
        <failure type="PHPUnit_Framework_AssertionFailedError">Lapangan_test::test_login_sukses
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php:99
</failure>
      </testcase>
      <testcase name="test_login_gagal" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="101" assertions="3" time="0.113434"/>
      <testcase name="test_login_gagal_password" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="110" assertions="1" time="0.090479">
        <failure type="PHPUnit_Framework_ExpectationFailedException">Lapangan_test::test_login_gagal_password
URL to redirect is not Redirect to http://localhost/GIT/index.php/lapangan/index/fail but Redirect to http://localhost/GIT/lapangan/login.
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-Redirect to http://localhost/GIT/index.php/lapangan/index/fail
+Redirect to http://localhost/GIT/lapangan/login

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:427
C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php:116
</failure>
      </testcase>
      <testcase name="test_login_gagal_username" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="120" assertions="1" time="0.119606">
        <failure type="PHPUnit_Framework_ExpectationFailedException">Lapangan_test::test_login_gagal_username
URL to redirect is not Redirect to http://localhost/GIT/index.php/lapangan/index/fail but Redirect to http://localhost/GIT/lapangan/login.
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-Redirect to http://localhost/GIT/index.php/lapangan/index/fail
+Redirect to http://localhost/GIT/lapangan/login

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:427
C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php:126
</failure>
      </testcase>
      <testcase name="test_APPPATH" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="156" assertions="1" time="0.052439"/>
      <testcase name="test_logout" class="Lapangan_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php" line="167" assertions="2" time="0.204640">
        <failure type="PHPUnit_Framework_ExpectationFailedException">Lapangan_test::test_logout
URL to redirect is not Redirect to http://localhost/GIT/index.php/lapangan but Redirect to http://localhost/GIT/lapangan/login.
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-Redirect to http://localhost/GIT/index.php/lapangan
+Redirect to http://localhost/GIT/lapangan/login

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:427
C:\xampp\htdocs\GIT\application\tests\controllers\Lapangan_test.php:177
</failure>
      </testcase>
    </testsuite>
    <testsuite name="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" fullPackage="Welcome" tests="22" assertions="10" failures="6" errors="0" time="3.814343">
      <testcase name="test_index" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="19" assertions="0" time="0.137607"/>
      <testcase name="test_ceklogin" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="25" assertions="1" time="0.162254"/>
      <testcase name="test_ceklogin_admin_not_login_kosongsemua" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="35" assertions="1" time="0.141262"/>
      <testcase name="test_ceklogin_admin_not_login_nopassword" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="45" assertions="1" time="0.145350"/>
      <testcase name="test_ceklogin_admin_not_login_nousername" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="55" assertions="1" time="0.154788"/>
      <testcase name="test_ceklogin_admin_not_login_unmatch" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="65" assertions="1" time="0.164850"/>
      <testcase name="test_login" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="75" assertions="0" time="0.156831"/>
      <testcase name="test_logout" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="80" assertions="1" time="0.209883">
        <failure type="PHPUnit_Framework_AssertionFailedError">Welcome_test::test_logout
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php:89
</failure>
      </testcase>
      <testcase name="test_inputlapangan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="92" assertions="1" time="0.215837"/>
      <testcase name="test_no_inputlapangan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="104" assertions="0" time="0.135190"/>
      <testcase name="test_dahboard" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="109" assertions="0" time="0.158263"/>
      <testcase name="test_datapenyewaan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="114" assertions="0" time="0.239954"/>
      <testcase name="test_tambahlapangan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="119" assertions="0" time="0.125343"/>
      <testcase name="test_editData" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="124" assertions="0" time="0.176164"/>
      <testcase name="test_datalapangan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="129" assertions="1" time="0.229045"/>
      <testcase name="test_not_datalapangan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="141" assertions="0" time="0.122441"/>
      <testcase name="test_deleteData" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="146" assertions="1" time="0.322879">
        <failure type="PHPUnit_Framework_AssertionFailedError">Welcome_test::test_deleteData
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php:155
</failure>
      </testcase>
      <testcase name="test_deleteDataLapangan" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="158" assertions="0" time="0.190685">
        <failure type="PHPUnit_Framework_AssertionFailedError">Welcome_test::test_deleteDataLapangan
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php:160
</failure>
      </testcase>
      <testcase name="test_validasi" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="163" assertions="0" time="0.138511">
        <failure type="PHPUnit_Framework_AssertionFailedError">Welcome_test::test_validasi
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php:165
</failure>
      </testcase>
      <testcase name="test_no_validasi" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="168" assertions="0" time="0.141694">
        <failure type="PHPUnit_Framework_AssertionFailedError">Welcome_test::test_no_validasi
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php:170
</failure>
      </testcase>
      <testcase name="test_do_editData" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="173" assertions="1" time="0.188169"/>
      <testcase name="test_do_editData_failed" class="Welcome_test" file="C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php" line="185" assertions="0" time="0.157343">
        <failure type="PHPUnit_Framework_AssertionFailedError">Welcome_test::test_do_editData_failed
redirect() is not called.

C:\xampp\htdocs\GIT\application\tests\_ci_phpunit_test\CIPHPUnitTestCase.php:407
C:\xampp\htdocs\GIT\application\tests\controllers\Welcome_test.php:188
</failure>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
